### ISSUE TITLE
Apollo 5.5 RTX 2080 Perception Support

### ISSUE THREAD

**Issue by lucasnmatias** [ApolloAuto/apollo#12710]
## System information

- **OS Platform and Distribution (e.g., Linux Ubuntu 18.04)**: Ubuntu 18.04
- **Apollo installed from (source or binary)**: Source
- **Apollo version (3.5, 5.0, 5.5, 6.0)**: 5.5
- **Output of `apollo.sh config` if on `master` branch**:

Hi, I've been working with Apollo 5.5, setting up everything to test the perception module using LGSVL. From documentation and issues I've notice that until recently the nvidia RTX 2080 was not supported on Apollo, leading the perception module to crash (exactly what happened to me). From more recents discussions it seems that RTX 2080 is now supported by Apollo but only on release 6.0. Since on the Apollo 6.0 the camera perception is not working I want to keep with Apollo 5.5, which is more stable and have the camera perception working. I would like to know if the RTX 2080 is also supported on Apollo 5.5 or if not, what would be necessary to make it supported.
Since on Apollo 6.0 the camera perception does not work I need to keep with Apollo 5.5 due to camera importance on my current project.

**Comment by oska874** [comment]
We have run Apollo  success on rtx 2080. The question is not the gpu but camera perception  itself.

**Comment by lucasnmatias** [comment]
@oska874 so at the moment it is not possible to run the camera perception on rtx 2080 right?

**Comment by oska874** [comment]
As our experience no matter which gou you use ,camera perception need to be small modify. And 2080 is suitable for apollo.

**Comment by lucasnmatias** [comment]
I've tested the Apollo 5.5 camera perception with nvidia 1080 and it worked, but when I tried to use nvidia 2080 it crashes.

**Comment by oska874** [comment]
Ohoh, we test apollo 5.5 with 2080 with newest nvidia driver and cuda 11. It's okay.

**Comment by storypku** [comment]
Hi @lucasnmatias ,

   Sorry for my late reply.

   This issue is about Nvidia GPU support capability of the Apollo project. 

In Apollo 5.5, only GPU cards with limited set of compute capabilities are supported, as can be seen below:

https://github.com/ApolloAuto/apollo/blob/r5.5.0/tools/cuda_library.bzl#L12

So,  GeForce GTX 1080 cards (SM=6.1) was supported in Apollo 5.5, while Geforce RTX 2080 cards (SM=7.5) was not fully supported: Maybe it runs in a backward-compatibility mode? 

To enhance Apollo's capability to support more recent GPU cards, we have made breaking changes in Apollo 6.0. @jeroldchen @panfengsu Do you have any updates on camera support since Apollo 6.0 release?

You can try to modify  `tools/cuda_library.bzl` in Apollo 5.5 to see if `sm=7.5` can work:

```
 "-gencode=arch=compute_61,code=sm_61",
"-gencode=arch=compute_75,code=sm_75",
 "-gencode=arch=compute_75,code=compute_75",
```

Most probably this can't work, and there seems no easy way to fix this issue in Apollo 5.5.
